services/movie.py

```diff
@@ -28,13 +33,15 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Ensure that `genres_ids` and `actors_ids` are always lists. You might want to initialize them as empty lists if they are `None` to avoid potential errors when calling `set` on them.